enhancements/installer/component-selection.md

```diff
@@ -386,12 +386,23 @@ Would expect this to go directly to GA once a design is agreed upon/approved.
 
 ### How To Implement A New Capability
 
-1. [Introduce](https://github.com/openshift/api/pull/1150) the new capablity name in the openshift/api repo
-    1. If no [versioned capability set](https://github.com/openshift/api/blob/8324d657dee1d594a8a7768e5569fea6a8f887a9/config/v1/types_cluster_version.go#L257-L291) exists for the current OCP version under development, introduce one as part of your PR.
-2. [Bump](https://github.com/openshift/cluster-version-operator/pull/737) the openshift/api vendored dependency in the openshift/cluster-version-operator repo
-3. [Bump](https://github.com/openshift/installer/pull/5645) the openshift/api vendoered dependency in the openshift/installer repo
-4. [Annotate](https://github.com/openshift/cluster-samples-operator/pull/414) your resources to use the new capability name so they can be excluded when your capability is disabled
-
+1. Select a name for the new capability, like `insights`.
+2. Open a pull request adding `capability.openshift.io/name` annotations to the manifests that deliver the capability, like [this][bump-annotations].
+    Some manifests should only be included if multiple capabilities are enabled, and in that case use the `+` delimiter between capability names, like `capability.openshift.io/name=insights+monitoring`.
+3. In presubmit CI for the pull request, the cluster-version operator will [exclude manifests with unrecognized capabilities][cvo-exclude-unrecognized].
+    Review presubmit results for your annotation pull request to ensure that:
+    1. The capability is being completely removed, without leaving dangling resources that you forgot to annotate.
+    2. CI suites which you expected to pass continue to pass.
+        You may find that some test-cases assume or require your capabilities presence, and they may need to grow logic to skip or alter the test conditions when your capability is not installed (like [this][bump-origin]).
+    If your annotation addition spands multiple pull requests, either because the manifests being annotated span multiple repositories or because you also need to make test suite adjustments in other repositories), you may be able to [use cluster-bot][cluster-bot] to run tests on a release assembling multiple in-flight pull requests.
```

Comment:
```suggestion
    If your annotation addition spans multiple pull requests, either because the manifests being annotated span multiple repositories or because you also need to make test suite adjustments in other repositories), you may be able to [use cluster-bot][cluster-bot] to run tests on a release assembling multiple in-flight pull requests.
```

Reply:
Fixed in 40652c8d -> 03ed3ad7.